plugins/auto-sizes/tests/improve-sizes-test.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Tests for the improve sizes for Images.
+ *
+ * @package auto-sizes
+ * @group   improve-sizes
+ */
+
+class Tests_Improve_Sizes extends WP_UnitTestCase {
+
+	/**
+	 * Set up the environment for the tests.
+	 */
+	public function setUp(): void {
+		parent::setUp();
+
+		switch_theme( 'twentytwentyfour' );
+	}
+
+	/**
+	 * Test the function with full alignment.
+	 *
+	 * @covers ::auto_sizes_improve_image_sizes_attribute
+	 */
+	public function test_full_alignment(): void {
+		$content      = '<img src="test.jpg" />';
```

Comment:
This works, but it is probably more accurate to try to crate content that actually mimics what the expected markup will be for an image block. You can review how I've set up fixtures in the `Test_AutoSizes` class to generate a test attachment and get the expected markup to test against.

Reply:
Thanks @joemcgill. Unit tests updated in https://github.com/WordPress/performance/pull/1250/commits/a0e4e264c046b169d9921a6bc902b6ab14251e7e